analytics_data_api/v0/tests/views/test_course_summaries.py

```diff
@@ -159,3 +163,10 @@ def test_programs(self):
         response = self.authenticated_get(self.path(exclude=self.always_exclude[:1], programs=['True']))
         self.assertEquals(response.status_code, 200)
         self.assertItemsEqual(response.data, self.all_expected_results(programs=True))
+
+    @ddt.data('passing_users', )
+    def test_exclude(self, field):
+        self.generate_data()
+        response = self.authenticated_get(self.path(exclude=[field]))
+        self.assertEquals(response.status_code, 200)
+        self.assertEquals(str(response.data).count(field), 0)
```

Comment:
`self.assertNotIn(field, response.data)` only works for the top-level (cumulative) field, not the per-mode field. This ensures the field is not present anywhere.